### ISSUE TITLE
Additional types in exports

### ISSUE THREAD

**Issue by dartess** [fingerprintjs/BotD#121]
HI! Can you please export `BotDetectionResult` from package?

Now the type can also be obtained, but this can only be done with a terrible construction like this:

```
type BotDetectionResult = ReturnType<
  (ReturnType<typeof load> extends Promise<infer T> ? T : never)['detect']
>;
```

**Comment by xnerhu** [comment]
Hi! Good catch, we will publish this feature soon.

**Comment by xnerhu** [comment]
Published in 1.5.0.
